stanford/playground/group_vars/all

```diff
@@ -23,6 +23,7 @@ EDXAPP_FEATURES_ENABLE_COURSEWARE_SEARCH: true
 EDXAPP_FEATURES_ENABLE_DASHBOARD_SEARCH: true
 EDXAPP_LMS_BASE: "{{ EDXAPP_SITE_NAME }}"
 EDXAPP_LMS_ENABLE_BASIC_AUTH: True
+EDXAPP_LMS_HTTPS: 'off'
```

Comment:
This will _not_ break in devstack because it's explicitly overridden in `devstack.py` 👍